install_microshift.py

```diff
@@ -0,0 +1,148 @@
+import host
+import subprocess
+import time
+
+def microshift_iso_builder() -> str:
+    lh = host.LocalHost()
+    cmd = f"dnf install -y osbuild-composer composer-cli cockpit-composer bash-completion firewalld"
+    result=lh.run(cmd)
+    logger.info(result)
+
+    cmd = f"systemctl enable osbuild-composer.socket cockpit.socket --now" 
```

Comment:
Should we clean up these instances as part of the cleanup phase of CDA?

Reply:
this isn't idempotent.